framework/src/auxkernels/MaterialStdVectorAux.C

```diff
@@ -34,5 +37,8 @@ MaterialStdVectorAux::~MaterialStdVectorAux()
 Real
 MaterialStdVectorAux::getRealValue()
 {
+  mooseAssert(_special_qp < _q_point.size(), "MaterialStdVectorAux.  special_qp specified as " << _special_qp << " but there are only " << _q_point.size() << " quadpoints in the element");
```

Comment:
Aha, i hadn't thought of that.  So if _special_qp >= _q_point.size() , do you think i should just return the value at qp=0 ?